### ISSUE TITLE
Not all values work correct

### ISSUE THREAD

**Issue by MikeyMan83** [MarFanNL/HeishamonMQTT#14]
Got an issue with some of the values not being presented in the graphs. For instance heat energy produced. It does show current value, but the graph stays at 0.

This happens with quite a few.

**Comment by MikeyMan83** [comment]
@MarFanNL @IgorYbema Any way you guys can have a quick look?

**Comment by IgorYbema** [comment]
Check if heishamon gets the values. And then if your mqtt server is receiving it.

**Comment by MikeyMan83** [comment]
@IgorYbema yes and yes. I see current value, but somehow the historic graphs are not filled

**Comment by IgorYbema** [comment]
Are the values updated in heishamon and mqtt?

**Comment by MikeyMan83** [comment]
Yes, even in domoticz. But no graphs.

**Comment by IgorYbema** [comment]
Seems like a domoticz issue then to me. I'm not using this plugin from @MarFanNL anymore so I can't help furhter.
If the pluging provides the data domoticz should then graph it.
